### ISSUE TITLE
Error when try to build the "slim" runtime

### ISSUE THREAD

**Issue by omerb01** [pywren/pywren-ibm-cloud#212]
I get errors by using `pywren-runtime build omerb01/pywren-omer-runtime:3.6.7` with the "slim" Docker image
```
omerbelh-mac:pywren-ibm-cloud omerbelh$ pywren-runtime build omerb01/pywren-omer-runtime:3.6.7
DEBUG:pywren_ibm_cloud.config:Loading configuration
DEBUG:pywren_ibm_cloud.config:Loading Compute backend module: ibm_cf
DEBUG:pywren_ibm_cloud.config:Loading Storage backend module: ibm_cos
DEBUG:pywren_ibm_cloud.compute.backends.ibm_cf.ibm_cf:Creating IBM Cloud Functions client
DEBUG:pywren_ibm_cloud.compute.backends.ibm_cf.ibm_cf:Set IBM CF Namespace to pywren_dev_us_east
DEBUG:pywren_ibm_cloud.compute.backends.ibm_cf.ibm_cf:Set IBM CF Endpoint to https://us-east.functions.cloud.ibm.com
DEBUG:pywren_ibm_cloud.compute.backends.ibm_cf.ibm_cf:IBM CF client created successfully
INFO:pywren_ibm_cloud.compute.backends.ibm_cf.ibm_cf:Creating a new docker image from Dockerfile
INFO:pywren_ibm_cloud.compute.backends.ibm_cf.ibm_cf:Docker image name: omerb01/pywren-omer-runtime:3.6.7
Sending build context to Docker daemon  4.913MB
[WARNING]: Empty continuation line found in:
    RUN pip install --upgrade setuptools six     && pip install --no-cache-dir         simplejson==3.16.0         httplib2==0.12.1         kafka_python==1.4.4         lxml==4.3.1         python-dateutil==2.7.5           pika==0.13.1            flask==1.1.1            gevent==1.4.0            ibm-cos-sdk==2.5.3      redis==3.3.8            requests        numpy==1.17.2 RUN mkdir -p /action
[WARNING]: Empty continuation lines will become errors in a future release.
Step 1/8 : FROM python:3.6-slim-buster
 ---> 7ccbe545a2f1
Step 2/8 : ENV FLASK_PROXY_PORT 8080
 ---> Using cache
 ---> 0d73a5a58ab7
Step 3/8 : RUN pip install --upgrade setuptools six     && pip install --no-cache-dir         simplejson==3.16.0         httplib2==0.12.1         kafka_python==1.4.4         lxml==4.3.1         python-dateutil==2.7.5            pika==0.13.1            flask==1.1.1     gevent==1.4.0           ibm-cos-sdk==2.5.3      redis==3.3.8            requests        numpy==1.17.2 RUN mkdir -p /action
 ---> Running in 084a3cb4de71
Collecting setuptools
  Downloading https://files.pythonhosted.org/packages/6a/9a/50fadfd53ec909e4399b67c74cc7f4e883488035cfcdb90b685758fa8b34/setuptools-41.4.0-py2.py3-none-any.whl (580kB)
Collecting six
  Downloading https://files.pythonhosted.org/packages/73/fb/00a976f728d0d1fecfe898238ce23f502a721c0ac0ecfedb80e0d88c64e9/six-1.12.0-py2.py3-none-any.whl
Installing collected packages: setuptools, six
  Found existing installation: setuptools 41.2.0
    Uninstalling setuptools-41.2.0:
      Successfully uninstalled setuptools-41.2.0
Successfully installed setuptools-41.4.0 six-1.12.0

Usage:   
  pip install [options] <requirement specifier> [package-index-options] ...
  pip install [options] -r <requirements file> [package-index-options] ...
  pip install [options] [-e] <vcs project url> ...
  pip install [options] [-e] <local project path> ...
  pip install [options] <archive url/path> ...

no such option: -p
The command '/bin/sh -c pip install --upgrade setuptools six     && pip install --no-cache-dir         simplejson==3.16.0         httplib2==0.12.1         kafka_python==1.4.4         lxml==4.3.1         python-dateutil==2.7.5           pika==0.13.1            flask==1.1.1             gevent==1.4.0           ibm-cos-sdk==2.5.3      redis==3.3.8            requests        numpy==1.17.2 RUN mkdir -p /action' returned a non-zero code: 2
```

**Comment by JosepSampe** [comment]
@omerb01 Fixed in #213

**Comment by omerb01** [comment]
@JosepSampe Thanks, tested and it works.
